code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -508,9 +508,6 @@
 	icon_state = "hair_manbun"
 	gender = MALE
 
-/datum/sprite_accessory/hair/bald
```

Comment:
RIP Bald?

Reply:
nah, that was a test copy I made while fiddling

Reply:
bald is redefined above, so that's weird

Reply:
One of the tests I had done was to make a version of "bald" that was explicitly insectoid only, before I realized there was nothing wrong with the hair. Bald is still there, and works fine.